doc/source/reference/algorithms.centrality.rst

```diff
@@ -57,17 +57,15 @@ Eigenvector
    katz_centrality
    katz_centrality_numpy
 
-Communicability
+Subgraph
 ---------------
```

Comment:
The underline has to match the length of the word for it to format without complaining.